### PR TITLE
Store: Settings: Payments: DRY payment sections

### PR DESCRIPTION
**Pull request by allendav**
We have a bit of duplication between the three payment sections. This DRYs that up.

To test:
* Ensure in Store > Settings > Payments you can still enable and manage on-site, off-site and offline payment methods

In support of #17271

### CODE REVIEW

File: client/extensions/woocommerce/app/settings/payments/index.js [hunk 1]
```diff
@@ -92,6 +91,45 @@ class SettingsPayments extends Component {
 		this.props.createPaymentSettingsActionList( successAction, failureAction );
 	};
 
+	renderPaymentSection = paymentSection => {
```
Comment (budzanowski) [inline client/extensions/woocommerce/app/settings/payments/index.js]: You could consider destructuring on the spot `... = ( { ... } ) =>'

File: client/extensions/woocommerce/app/settings/payments/index.js [hunk 2]
```diff
@@ -92,6 +91,45 @@ class SettingsPayments extends Component {
...
+		return (
```
Comment (budzanowski) [inline client/extensions/woocommerce/app/settings/payments/index.js]: Change the `=> { ` to ` `=> (` and you don't need the `return`

**Comment by budzanowski** [comment]
Small ( very ) nits. Good to go.

File: client/extensions/woocommerce/app/settings/payments/index.js (hunk 1 above)
Comment (allendav) [reply client/extensions/woocommerce/app/settings/payments/index.js]: Oooooh!

File: client/extensions/woocommerce/app/settings/payments/index.js (hunk 2 above)
Comment (allendav) [reply client/extensions/woocommerce/app/settings/payments/index.js]: Good idea - will do!

Comment (allendav) [reply client/extensions/woocommerce/app/settings/payments/index.js]: Fixed in https://github.com/Automattic/wp-calypso/pull/20928/commits/71969111fdc801cecfb52fd07ec0c7b9086a1afe

File: client/extensions/woocommerce/app/settings/payments/index.js (hunk 1 above)
Comment (allendav) [reply client/extensions/woocommerce/app/settings/payments/index.js]: Fixed in https://github.com/Automattic/wp-calypso/pull/20928/commits/71969111fdc801cecfb52fd07ec0c7b9086a1afe

**Pull request merged**